index.js

```diff
@@ -1,13 +1,14 @@
 var relative = require('relative-date')
 var pretty = require('prettier-bytes')
 var path = require('path')
-var data = require('render-data')
 var yo = require('yo-yo')
+var data = module.parent ? null : require('render-data')
 
 module.exports = Tree
 
 function Tree (root, entries, onclick) {
   if (!(this instanceof Tree)) return new Tree(root, entries, onclick)
+  if (module.parent) onclick = function () {}
```

Comment:
when you `require('yo-fs')`, `module.parent` is always going to exist, no matter if node, browser, electron, etc. This pattern only works for https://github.com/datproject/dat.land/blob/master/client/js/app.js because in the case of client side execution, this file is the entry point of everything, thus `module.parent` doesn't exist.

You're right that in electron `process.browser` doesn't exist (because it's a browserify thing), so it's not a good check. `typeof window` should work though, although it's better not to check the environment at all inside a module.